src/www/ui/ui-view-info.php

```diff
@@ -539,28 +540,29 @@ function ShowReportInfo($Upload)
       $gaSelectionList = explode(',', $row['ri_ga_checkbox_selection']);
     }
 
-    $vars['footerNote']           = $footerNote;
-    $vars['reviewedBy']           = $reviewedBy;
-    $vars['reportRel']            = $reportRel;
-    $vars['community']            = $community;
-    $vars['component']            = $component;
-    $vars['version']              = $version;
-    $vars['relDate']              = $relDate;
-    $vars['sw360Link']            = $sw360Link;
-    $vars['generalAssesment']     = $generalAssesment;
-    if (array_key_exists(8, $gaSelectionList)) {
-      $vars['nonCritical']        = $gaSelectionList[0];
-      $vars['critical']           = $gaSelectionList[1];
-      $vars['noDependency']       = $gaSelectionList[2];
-      $vars['dependencySource']   = $gaSelectionList[3];
-      $vars['dependencyBinary']   = $gaSelectionList[4];
+    $vars['footerNote'] = $footerNote;
+    $vars['reviewedBy'] = $reviewedBy;
+    $vars['reportRel'] = $reportRel;
+    $vars['community'] = $community;
+    $vars['component'] = $component;
+    $vars['version'] = $version;
+    $vars['relDate'] = $relDate;
+    $vars['sw360Link'] = $sw360Link;
+    $vars['generalAssesment'] = $generalAssesment;
+    if (array_key_exists(9, $gaSelectionList)) {
+      $vars['nonCritical'] = $gaSelectionList[0];
+      $vars['critical'] = $gaSelectionList[1];
+      $vars['noDependency'] = $gaSelectionList[2];
+      $vars['dependencySource'] = $gaSelectionList[3];
+      $vars['dependencyBinary'] = $gaSelectionList[4];
       $vars['noExportRestriction'] = $gaSelectionList[5];
-      $vars['exportRestriction']  = $gaSelectionList[6];
-      $vars['noRestriction']      = $gaSelectionList[7];
-      $vars['restrictionForUse']  = $gaSelectionList[8];
+      $vars['exportRestriction'] = $gaSelectionList[6];
+      $vars['noRestriction'] = $gaSelectionList[7];
+      $vars['restrictionForUse'] = $gaSelectionList[8];
+      $vars['spdxLicenseComment'] = $gaSelectionList[9];
```

Comment:
probably this is not the right place to add the spdx Comment,
Better to create a new field in report_info table, and create a separate table in the UI to show spdx license comment.